.github/workflows/e2e_scripts/clean.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set +e
+
+# This script is responsible for deleting resources out of a
+# stackdriver sandbox project in between test runs
+
+export PROJECT_ID=$(gcloud config get-value project)
+
+# delete cluster
+CLUSTER_ZONE="first_run"
+while [ -n "$CLUSTER_ZONE" ]; do
+  CLUSTER_ZONE=$(gcloud container clusters list \
+                   --filter="name:stackdriver-sandbox" \
+                   --project $PROJECT_ID --format="value(zone)")
+  if [ -n "$CLUSTER_ZONE" ]; then
+      echo "deleting cluster"
+      gcloud container clusters delete stackdriver-sandbox \
+          --project $PROJECT_ID --zone $CLUSTER_ZONE --quiet
+      sleep 20
+  fi
+done
+
+
+# delete loadgenerator
+LOADGENRATOR="first_run"
+while [ -n "$LOADGENRATOR" ]; do
```

Comment:
Maybe worth moving this script to be in the same place where destroy.sh is. This way we might even offer users the ability to clean the project rather than destroy it in the future.

Reply:
Also, making changes in provisioning will be easier - less change we'll miss appropriate updates that also need to be done in clean.sh

Reply:
I wrote this for test automation in mind, it isn't currently safe for user input. Right now, it will automatically wipe the currently active project (even if it isn't a sandbox project!). If we move it to the user-facing folder, I think I should add some prompts and guardrails first. Do you think that's worth doing?

Reply:
Let's keep it for now in the test automation folder but we need to be extra careful in code reviews - if we add more objects that needs to be provisioned, we also need to handle their cleanup here.